src/promptflow/promptflow/batch/_batch_engine.py

```diff
@@ -160,8 +159,8 @@ def run(
                 try:
                     # register signal handler for python flow in the main thread
                     # TODO: For all executor proxies that are executed locally, it might be necessary to
-                    # register a signal for Ctrl+C in order to customize some actions beyond just killing
-                    # the process, such as terminating the executor service.
+                    #  register a signal for Ctrl+C in order to customize some actions beyond just killing
+                    #  the process, such as terminating the executor service.
```

Comment:
Extra space

Reply:
this is still the TODO sentence, so need an extra space to let IDE know this
![image](https://github.com/microsoft/promptflow/assets/37076709/e2973b2f-d41a-412d-af82-990b2bb57e59)
![image](https://github.com/microsoft/promptflow/assets/37076709/ac1f0178-3f32-455e-9d09-4cc325bc18bb)